### DISCUSSION TITLE
Multibranch webhook trigger plugin issue in jenkins

### DISCUSSION THREAD

**Discussion by ghost** [learndevopsonline/learndevopsonline#443] [Q&A] [answered]
Hi Raghu. I have installed multibranch webhook trigger plugin and set a token as cart in the configuration and also in gitlab i have set the webhook url. Everything is fine, im able to trigger the url but when i trigger there is no builld happening in jenkins cart project but the log shows theres a trigger. 
repo url is https://gitlab.com/msamrat789/cart 
Also mentioned the ip screen shot of jenkins server
<img width="914" alt="Screenshot 2023-04-07 135804" src="https://user-images.githubusercontent.com/54846444/230573711-93201527-ad6d-4697-817d-2b747cd6fdae.png">
<img width="492" alt="Screenshot 2023-04-07 135851" src="https://user-images.githubusercontent.com/54846444/230573724-342aa8c8-862d-4eed-b0e8-f7f008f1169e.png">

**Comment by chandrashekarsubuddi** [comment, marked as answer]
as per my understanding May be you need to review the jenkins configuration. discover tags. if code is older than 7 days it wont trigger respected cart jenkins file not updated

**Reply by ghost** [reply]
I have checked the configuration , everything is fine. When i create multibranch pipeline as test seperatly, the job is running when there is some change but if i create the pipeline inside a folder it is detecting the change but not trying to auto build the job. u can see the log below in the pic
<img width="621" alt="Screenshot 2023-04-07 170536" src="https://user-images.githubusercontent.com/54846444/230602350-90835e2e-f1b0-495a-b39f-59c15629db26.png">
